static/js/containers/SubscriptionsSidebar.js

```diff
@@ -0,0 +1,55 @@
+// @flow
+import React from "react"
+import { connect } from "react-redux"
+import { Link } from "react-router-dom"
+import type { Dispatch } from "redux"
+
+import { channelURL } from "../lib/url"
+import { actions } from "../actions"
+import type { RestState } from "../flow/restTypes"
+import type { Channel } from "../flow/discussionTypes"
+
+type SubscriptionsSidebarProps = {
+  subscriptions: RestState<Array<Channel>>,
+  dispatch: Dispatch
+}
+
+class SubscriptionsSidebar extends React.Component {
+  props: SubscriptionsSidebarProps
+
+  componentWillMount() {
+    const { subscriptions, dispatch } = this.props
+    if (!subscriptions.loaded && !subscriptions.processing) {
+      dispatch(actions.subscriptions.get())
+    }
+  }
+
+  render() {
+    const { subscriptions } = this.props
+
+    if (!subscriptions.data) {
+      return null
+    }
+
+    return (
+      <div className="subscriptions">
+        {subscriptions.data.map(channel =>
+          <div key={channel.name}>
+            <Link to={channelURL(channel.name)}>
+              {channel.title}
+            </Link>
+          </div>
+        )}
+      </div>
+    )
+  }
+}
+
+const mapStateToProps = state => {
+  return {
+    channels:      state.channels,
+    subscriptions: state.subscriptions
+  }
+}
+
+export default connect(mapStateToProps)(SubscriptionsSidebar)
```

Comment:
I'm confused about a few things regarding this container. 

1. `state.channels` and `state.subscriptions` - Shouldn't `state.channels` be `state.channelDetail` or something like that? It looks like it only represents the endpoint to get/create a single channel. `channelsEndpoint` and other similar variables should also be renamed. I also think `state.subscriptions` should probably be `state.channels` even though at the moment I guess it's only pulling subscribed channels. It's using the `/channels` endpoint, right? I could be convinced otherwise.
1. I'm not sure we've ever followed a pattern where we render a container from another container. I think this should be a component with the subscribed channels passed in as a prop. It seems perfectly reasonable to do the work of fetching subscribed channels in `ChannelsPage`